meta/group-info/zh.yaml

```diff
@@ -13,28 +12,28 @@ gmt:
   all_mods: 全局管理员
   areas:
     beatmap: 谱面管理员
-    chat: 聊天室管理员
+    chat: 聊天管理员
     forum: 论坛管理员
     team: 官方团队
-    dev: osu! 开发
+    dev: osu! 开发团队
```

Comment:
the english version is "osu! development", so wasn't this correct already?

Reply:
in Chinese, the translation of `development` is a verb, so I added a noun to make it more accurate

Reply:
it's also the reason for the following strings (player, support) modification.

Reply:
here it's a slight problem since "osu! development" doesn't mean being part of the dev team

Reply:
Does this term `osu! development` represents a group of contributors with responsibilities (BNs etc), rather than a group of people working around ppy and getting paid?

Reply:
it represents people who contribute to the development of osu!, while `alumni.roles.dev` ("osu!dev") represents people who have been in the [`DEV` user group](https://osu.ppy.sh/groups/11) (the development team), at least from what i can deduce

Reply:
ok